packages/ubuntu_bootstrap/lib/services/identity_service.dart

```diff
@@ -1,14 +1,18 @@
+import 'dart:convert';
+
 import 'package:crypt/crypt.dart';
 import 'package:meta/meta.dart';
 import 'package:subiquity_client/subiquity_client.dart';
 import 'package:ubuntu_bootstrap/services/post_install_service.dart';
 import 'package:ubuntu_provision/services.dart';
 
 class SubiquityIdentityService implements IdentityService {
-  const SubiquityIdentityService(this._subiquity, this._postInstall);
+  const SubiquityIdentityService(this._subiquity, this._postInstall,
+      {@visibleForTesting this.testSalt});
```

Comment:
Trailing comma

Reply:
Ah, we're still using `ubuntu_lints` 0.3.0 here.. thanks!